vm/vm_test.go

```diff
@@ -19,6 +19,11 @@ func TestVm(t *testing.T) {
 	pytest.RunTests(t, "tests")
 }
 
+func TestVm2(t *testing.T) {
+	// TODO(wetor) temp test
+	pytest.RunTests(t, "tests2")
```

Comment:
why the new `tests2` directory ?

Reply:
This is a temporary debug code, and I will eventually remove it